app/brave_generated_resources.grd

```diff
@@ -162,15 +162,28 @@ By installing this extension, you are agreeing to the Google Widevine Terms of U
       <message name="IDS_PROFILES_OPEN_TOR_PROFILE_BUTTON" desc="Button in the avatar menu bubble view to open a Tor window.">
         Open Tor Window
       </message>
-      <message name="IDS_NEW_OFFTHERECORD_WINDOW_TOR" desc="The text label of a menu item to open a new off-the-record window with Tor.">
-        New private window with Tor
-      </message>
-      <message name="IDS_CONTENT_CONTEXT_OPENLINKTOR" desc="The name of the open a link in private window with Tor command">
-        Open link in private window with Tor
-      </message>
-      <message name="IDS_CONTENT_CONTEXT_OPENLINKTOR_INAPP" desc="The name of the open a link in private window with Tor command">
-        Open link in Brave private window with Tor
-      </message>
+      <if expr="use_titlecase">
+        <message name="IDS_NEW_OFFTHERECORD_WINDOW_TOR" desc="Title case: The text label of a menu item to open a new off-the-record window with Tor.">
+          New Private Window with Tor
+        </message>
+        <message name="IDS_CONTENT_CONTEXT_OPENLINKTOR" desc="Title case: The name of the open a link in private window with Tor command">
+          Open Link in Private Window with Tor
+        </message>
+        <message name="IDS_CONTENT_CONTEXT_OPENLINKTOR_INAPP" desc="Title case: The name of the open a link in private window with Tor command">
+          Open Link in Brave Private Window with Tor
```

Comment:
Not sure why we call out the app name here?
```suggestion
          Open Link in Private Window with Tor
```

Reply:
@tomlowenthal I'm copying what comes from Chromium. I'm guessing that this string is shown in an App / PWA window which does not have any tabs or toolbar showing, so the user may not even know it's in Brave. This wording let's the user know that it will open the link in a Brave private window and not in the current 'app' window.

Reply:
🤷🏻‍♂️ I'd like to be sure of that though.